setup.py

```diff
@@ -23,11 +23,11 @@
     ],
     packages=find_packages(exclude=['tests']),
     install_requires=[
-        'django>=1.8,<1.9',
+        'django>=1.8,<2.0',
         'django-model-utils>=1.4.0',
         'djangorestframework>=3.2.0,<3.4.0',
         'djangorestframework-oauth>=1.1.0,<2.0.0',
-        'edx-django-oauth2-provider>=0.5.0,<1.0.0',
+        'edx-django-oauth2-provider>=1.2.0',
```

Comment:
Without this change, version 0.5.0 was being installed in Travis and would error. Now it is installing 1.2.0 in Travis in passing.

Unfortunately locally it is still installing 0.5.0 and failing, and I haven't been able to figure out why.